docs/UserGuide.md

```diff
@@ -559,14 +559,31 @@ There are three `TYPE`s:
 
 `com` refers to Company tab, `app` refers to Application tab, `me` refers to Profile tab.
 
-Example: 
-* `switch me`
+Example:
+
+1. Suppose you want to switch to the application tab. Executing `switch app` will switch the tabs to application tab.
```

Comment:
Maybe you could change it to switch tabs to the application tab

Reply:
fixed